cmd/root.go

```diff
@@ -46,10 +46,8 @@ func Execute() {
 
 func init() {
 	persistent := rootCmd.PersistentFlags()
-	persistent.BoolP("disable_slack", "d", false, "Disable Slack alerts.")
-
 	persistent.StringP("config", "c", "config.toml", "Config file path.")
-
+	persistent.StringSliceP("reporters", "r", []string{"slack"}, "Specify a list of reporters for reporting vulnerabilities.")
```

Comment:
```suggestion
	persistent.StringSliceP("reporters", "r", []string{"slack", "console"}, "Specify a list of reporters for reporting vulnerabilities.")
```

We definitely want the console reporter enabled by default :smile:

Reply:
fixed